google/api_version_converters.go

```diff
@@ -0,0 +1,80 @@
+package google
+
+import (
+	"encoding/json"
+	"google.golang.org/api/compute/v1"
+)
+
+func convert(item, out interface{}) error {
+	bytes, err := json.Marshal(item)
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(bytes, out)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func convertInstanceGroupManagerToV1(self interface{}) (*compute.InstanceGroupManager, error) {
```

Comment:
As discussed offline, let's at least see what it looks like to do it without the separate functions.